1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,12 +21,36 @@ let basketballTeam = {
 */
 
 // write code here
+function getAcessAndSort(object, key2sort) {
```

Comment:
It's a good idea to generalise the code in order to make it work for future scenarios. But in this case you don't need to, so it's ok not to have `key2sort` and the `Array.isArray` check. You're allowed to assume the input object is valid.
However in a professional context this is a good thing to do, in moderation.